package.json

```diff
@@ -17,7 +17,7 @@
     "packages/*"
   ],
   "scripts": {
-    "prebuild": "lerna exec --scope @datapunt/asc-assets -- yarn build",
+    "prebuild": "scripts/port-check.sh && lerna exec --scope @datapunt/asc-assets -- yarn build",
```

Comment:
- port-check.sh not working on windows (lsof not known)
- in asc-assets/packakge.json there is commando mkdir-p lib that doesn't work. 
please replace with crossplatform commado's, or mention in de readme dat windows is not supported